README.md

```diff
@@ -84,7 +84,7 @@ These are the commands to run (taken from the above link):
 cd /srv/sites/documentation-shop.histwest.org.au
 find var vendor pub/static pub/media app/etc -type f -exec chmod g+w {} \;
 find var vendor pub/static pub/media app/etc -type d -exec chmod g+ws {} \;
-chown -R :<web server group> .
+chown -R :www-data .
```

Comment:
This was unnecessarily abstracted, it's always `www-data` (both apache and nginx).